docs/carbon-aware-webapi.md

```diff
@@ -312,15 +312,47 @@ The response is an array of CarbonIntensityDTO objects which each have a locatio
 ]
 ```
 
+## List of Locations
+
+Supported location codes by the API. Locations marked with \* return 204 (NO DATA).
```

Comment:
This is the locations of Azure Data centres as currently supported by the Azure Regions configured by the API, and the 204 data response is based on this being configured with the WattTime API.

Neither of these is going to be static over time with Azure and WattTime adding regions and region support respectively.

We also have ElectricityMap support in progress, which in turn means we end up with a complex matrix of supported region/source data provider combinations.

We will review in the weekly SDK call and think about an approach to document because it's useful, but maybe should be separate to the API documentation itself i.e. possibly we link off to an Azure document.

No action from us yet, just ensuring you see the comment :)